soundfiler: revert hint on avoiding BOM

ben-wes opened the pull request:
hint becomes obsolete with 43b4e8291a7d8f981a876a2526e532e49c0daeac and the "must" is factually wrong.

the other changes in the previous PR are still valid though - so i'm reverting the BOM hint with this additional commit.

ben-wes committed:
soundfiler: revert hint on avoiding BOM
not necessary anymore with commit 43b4e8291a7d8f981a876a2526e532e49c0daeac

also meant to close https://github.com/pure-data/pddp/issues/266

ben-wes commented:
not sure if we should do this - but imho, the hint is not required anymore then (and the "must" is factually wrong).

umlaeute commented:
i think this revert is probably overdoing things.

while it is true that with https://github.com/pure-data/pure-data/commit/43b4e8291a7d8f981a876a2526e532e49c0daeac the BOM is no more a problem, the rest still holds true: 
> The file must only contain ASCII characters

but anyhow, only include this changes once `develop` has been merged :-)

ben-wes commented:
> the rest still holds true

> The file must only contain ASCII characters

ok, i added that part back. my reasoning was that the other statements might suffice:
* "(read a file containing ascii numbers)"
* "Non-numeric fields get replaced by zero."

... but it's certainly good to mention it also in the paragraph.

updated this PR including changes addressing https://github.com/pure-data/pddp/issues/266

ben-wes commented:
@porres with https://github.com/pure-data/pure-data/commit/43b4e8291a7d8f981a876a2526e532e49c0daeac now merged, this PR should be ready to go.
